libs/gi/ui/src/components/character/CharacterMultiSelectionModal.tsx

```diff
@@ -0,0 +1,500 @@
+                database.displayCharacter.set({ ascending })
+              }
+            />
+          </Box>
+        </CardContent>
+        <Divider />
+        <DataContext.Provider value={{ teamData: undefined } as any}>
+          <CardContent sx={{ flex: '1', overflow: 'auto' }}>
+            <Grid
+              container
+              spacing={1}
+              columns={{ xs: 2, sm: 3, md: 4, lg: 5 }}
+            >
+              {characterKeyList.map((characterKey) => (
+                <Grid item key={characterKey} xs={1}>
+                  <SelectionCard
+                    characterKey={characterKey}
+                    onClick={() => onClick(characterKey)}
+                    selectedCharKeys={currentTeamCharKeys}
+                  />
+                </Grid>
+              ))}
+            </Grid>
+          </CardContent>
+        </DataContext.Provider>
+      </CardThemed>
+    </ModalWrapper>
+  )
+}
+
+const CustomTooltip = styled(({ className, ...props }: TooltipProps) => (
+  <Tooltip {...props} classes={{ popper: className }} />
+))({
+  [`& .${tooltipClasses.tooltip}`]: {
+    padding: 0,
+  },
+})
+
+function SelectionCard({
+  characterKey,
+  onClick,
+  selectedCharKeys,
+}: {
+  characterKey: CharacterKey
+  onClick: () => void
+  selectedCharKeys: (CharacterKey | '')[]
+}) {
+  const { gender } = useDBMeta()
+  const character = useCharacter(characterKey)
+  const { favorite } = useCharMeta(characterKey)
+  const database = useDatabase()
+  const { silly } = useContext(SillyContext)
+
+  const [open, onOpen, onClose] = useBoolState()
+
+  const { level = 1, ascension = 0, constellation = 0 } = character ?? {}
+  const banner = characterAsset(characterKey, 'banner', gender)
+  const rarity = getCharStat(characterKey).rarity
+
+  const isSelected = selectedCharKeys.indexOf(characterKey) !== -1
+  return (
+    <CustomTooltip
+      enterDelay={300}
+      enterNextDelay={300}
+      arrow
+      placement="bottom"
+      open={open}
+      onClose={onClose}
+      onOpen={onOpen}
+      title={
+        <Box sx={{ width: 300 }}>
+          <CharacterCard hideStats characterKey={characterKey} />
+        </Box>
+      }
+    >
+      <Box>
+        <CardThemed
+          bgt="light"
+          sx={{
+            position: 'relative',
+            flexGrow: 1,
+            display: 'flex',
+            flexDirection: 'column',
+            outline: isSelected
+              ? 'solid gold'
+              : undefined,
+          }}
+        >
+          <IconButton
+            sx={{ p: 0.25, position: 'absolute', zIndex: 2, opacity: 0.7 }}
+            onClick={(_) => {
+              onClose()
+              database.charMeta.set(characterKey, { favorite: !favorite })
+            }}
+          >
+            {favorite ? <FavoriteIcon /> : <FavoriteBorderIcon />}
+          </IconButton>
+          {isSelected && (
+            <Typography variant="body2" sx={{ flexGrow: 1 }}>
```

Comment:
This doesnt seem to update when you 
hover a character to see the tooltip
click
dont move the mouse

it will only start showing/hiding the border once i move my mouse away and it closes the tooltip

Reply:
Yeah, I noticed that locally too and wanted to make sure it was also happening in a live environment. Still need to figure out what's causing it, probably something's not updating when I think it is.

Reply:
yeah, its very common to be in a state where the select is not showing 
![image](https://github.com/user-attachments/assets/bad198e2-8588-4307-a480-7be3fe5e0bee)